src/buildercore/trop.py

```diff
@@ -542,6 +542,21 @@ def render_rds(context, template):
         data['StorageEncrypted'] = True
         data['KmsKeyId'] = lu('rds.encryption') if isinstance(lu('rds.encryption'), str) else ''
 
+    # use existing snapshot to create instance:
+    # - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-dbsnapshotidentifier
+    if lu('rds.snapshot-id', None):
+        data['DBSnapshotIdentifier'] = lu('rds.snapshot-id')
+        delete_these = [
+            # in use
+            "DBName", "MasterUsername",
```

Comment:
add note that even though the dbname is preserved in the buildvars it won't be available in the template (or even necessarily consistent with what is in the snapshot).